src/vite-env.d.ts

```
@@ -14,3 +14,58 @@ interface ImportMetaEnv {
+        MarkerOptions: {
+          map: kakao.maps.Map;
+          position: kakao.maps.LatLng;
+        };
+      };
+    };
+  }
+  
+  namespace kakao {
+    namespace maps {
+      class Map {
+        constructor(container: HTMLElement, options: MapOptions);
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)